iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -349,19 +348,30 @@ export class iXBRLViewer {
             const progress = stubViewer ? 'Loading iXBRL Report' : 'Loading iXBRL Viewer';
             iv.setProgress(progress).then(() => {
                 /* Poll for iframe load completing - there doesn't seem to be a reliable event that we can use */
-                let timer = setInterval(function () {
+                const timer = setInterval(() => {
                     let complete = true;
-                    iframes.each(function (n) {
-                        const iframe = this;
+                    iframes.each((n, iframe) => {
                         const iframeDoc = iframe.contentDocument || iframe.contentWindow.document;
-                        if ((iframeDoc.readyState != 'complete' && iframeDoc.readyState != 'interactive') || $(iframe).contents().find("body").children().length == 0) {
+                        if ((iframeDoc.readyState !== 'complete' && iframeDoc.readyState !== 'interactive') || $(iframe).contents().find("body").children().length === 0) {
                             complete = false;
                         }
                     });
                     if (complete) {
                         clearInterval(timer);
 
-                        const viewer = iv.viewer = new Viewer(iv, iframes, reportSet);
+                        iframes.each((n, iframe) => {
+                            const htmlNode = $(iframe).contents().find('html');
+                            if (htmlNode.attr('lang') === undefined) {
```

Comment:
This is fine, but this check should never return true since `lang` isn't valid iXBRL, right?  Should we just omit the check or add a comment why we're checking it?